tests/testsuite.py

```diff
@@ -699,3 +704,27 @@ def test_write(self):
         self.writer.flush()
         self.assertEqual(self.getFirstContent(), self.getSecondContent())
         self.assertEqual(wrote, len(self.getSecondContent()))
+
+
+@unittest.skipIf(sys.version_info[0] < 3, 'Python 3 required')
+class XMLProgramTestCase(unittest.TestCase):
+    @mock.patch('sys.argv', ['xmlrunner', '-o', 'flaf'])
+    @mock.patch('xmlrunner.runner.XMLTestRunner')
+    @mock.patch('sys.exit')
+    def test_xmlrunner_output(self, exiter, testrunner):
+        xmlrunner.runner.XMLTestProgram()
+
+        kwargs = dict(
+            buffer=False,
+            failfast=False,
+            verbosity=1,
+            warnings='default',
+            output='flaf',
+        )
+
+        if sys.version_info[:2] > (3, 4):
+            kwargs.update(tb_locals=False)
```

Comment:
unreachable?

Reply:
Erm, no? The line prior ensures its invocation on Python 3.5 and later?

Reply:
read 709 wrong :)